### DISCUSSION TITLE
Turbine yaml file FLORIS v4

### DISCUSSION THREAD

**Discussion by windexplorer21** [NatLabRockies/floris#886] [Q&A]
What are these coefficients present in the iea_15MW.yaml turbine file?
helix_a: 1.809
  helix_power_b: 4.828e-03
  helix_power_c: 4.017e-11
  helix_thrust_b: 1.390e-03
  helix_thrust_c: 5.084e-04
If I want to generate a new turbine file, how can I be sure they won't be considered?
Thank you in advance

**Comment by misi9170** [comment]
Hi @windexplorer21 ,

These parameters describe how the IEA 15MW behaves when operating with Helix active wake control (AWC). You can find details about that [here](https://nrel.github.io/floris/operation_models_user.html#awc-model). If you do not intend to run your turbine with Helix, you can simply leave these parameters out of your turbine yaml.

**Reply by windexplorer21** [reply]
Thank you so much! 🙏
